### ISSUE TITLE
Staff Area report

### ISSUE THREAD

**Issue by bethlakshmi** [bethlakshmi/GBE2#250]
This was a request from Seth last year, and it was quite good...

Each staff head would like to see a report (and probably print it, but it doesn't have to be as pretty as a public report) for their area.

For example, if I'm Betty and I'm running registration, I want to see that for my area:
- what volunteer opportunities I have set up
- what actual volunteers I have 
- sorted by time/day

So I'd see something like:

Registration:
- 8:30 - 12:30 - Head of reg desk - 1 max volunteer
  - Marek Rivero - marek@email.com
- 9:00-12:00 - Staff reg desk - 2 max volunteers, 
   - Velma the Volunteer - velma@volunteer.com
   - Vick the Volunteer - vick@volunteer.com
- 12:00-3:00 - Staff reg desk - 2 max volunteers
   - *no volunteers at this time*

And so forth.  "Head of reg desk" or "staff reg desk" would be titles of volunteer opportunities.  It'd be nice to have the type as well.

I don't think it matters if we have cross staff visibility, so long as it's only visible to staff.  It could chain off of the scheduled event editing pages in the current scheduler.

**Comment by burlexpo** [comment]
Also phone numbers. When someone doesn't show up for their shift, you really want to pull them more than you want to email them.

The Great Burlesque Exposition #9
February 20-22, 2015
www.Burlesque-Expo.com
#BurlExpo9

**Comment by jonkiparsky** [comment]
- do we want a formatted report, or is csv more useful?
it's certainly possible to generate, for example, a PDF, but that might
take a little botheration, and give the consumer of the report less
flexibility than they'd really want. I don't mind doing extra work, but not
if it produces less useful output.

- I'm assuming these are broken out by "Volunteer Interests" categories
("Volunteer Type" on the Manage Opportunities form), meaning we have
reports for Registration, Security, Costume Exhibit, Stage Management, etc.
Is this correct?

- I assume you mean placing it near the manage opportunities form? This
makes sense, except that those pages are event-specific, and this report is
not event-specific. Given the amount of reporting being requested, I think
we're starting to see a  single reporting page, with links to each of these
reports. (possibly filtered by permission groups, if we want to get fancy)

**Comment by bethlakshmi** [comment]
Per staff area - pick staff area first.

**Comment by bethlakshmi** [comment]
Answering Jon's questions:
- a csv version might be OK, but I don't feel strongly.  PDF is not that useful - I don't know that nice printability is worth the effort.  I just made rough version, and the HTML alone may be OK.

- I was thinking of it per staff area, not so much by type.  I'm assuming that a given staff may have several different types all within their staff area, and want the whole picture for the area.

- agree on the not-event specific - but I think each staff member may want their own event, and feedback from Seth last year was that he didn't want *everyone's* - it was a huge amount of data, he only wanted his.  So I made a landing page, with a table for the event, head of staff, and then a link to see the event's opportunities.

Closing, but glad to revisit tickets with commentary after we get this up to a site.
